### DISCUSSION TITLE
Hosting a react flask (async) web app on vercel?

### DISCUSSION THREAD

**Discussion by MonkeeMan1** [vercel/community#544] [Help] [answered]
Hey all.

Just wondering if its possible to host a flask web app which uses react on vercel? I have successfully setup the react front end but am unsure if/how I can also host the backend (flask async) on vercel too?

**Comment by leerob** [comment, marked as answer]
Hey! Unfortunately no, a flask server won't work on Vercel (a serverless platform).

**Reply by MonkeeMan1** [reply]
Alright, thank you very much for the response.
